rezerwacje-front/src/views/Users/components/UsersForm/UsersForm.js

```diff
@@ -0,0 +1,276 @@
+import React from 'react';
+import {
+  Card,
+  CardHeader,
+  CardContent,
+  CardActions,
+  Divider,
+  Grid,
+  Button,
+  TextField,
+  Typography,
+  Radio, 
+  RadioGroup,
+  FormControlLabel
+} from '@material-ui/core';
+import useForm from "./useForm"
+import validateUserForm from "./validateUserForm"
+import './Users.scss'
+
+const UsersForm = () => {
+
+   const initState = {
+    firstName: '',
+    lastName: '',
+    email: '',
+    phone: '',
+    login: '',
+    password: '',
+    confirmPassword: '',
+    role: ''
+  };
+
+
+  const submit = () => {
+
+      const {firstName, lastName, email, phone, login, password, role} = values
+      console.log(JSON.stringify({firstName: firstName, lastName: lastName, emailAddress: email, phoneNumber: phone, 
+        login: login,  password: password, userType: role}))
+      
+        fetch("/api/system-user", {
+          method: "POST", 
+          headers: { 'Content-Type': 'application/json' },
+          body: JSON.stringify({firstName: firstName, lastName: lastName, emailAddress: email, phoneNumber: phone, 
+      login: login,  password: password, userType: role}),
+        }).then(
+          function (res) {
+            if (res.ok) {
+              setState(initState);
+              console.log(values);
+              alert("Użytkownik został dodany do bazy.");
+            } else if (res.status === 400) {
+              alert("Wystąpił błąd.");
+            }
+          },
+          function (e) {
+            alert("Wystąpił błąd.");
+          }
+        );
+  }
+
+  const { handleChange, handleSubmit, values, errors, setState} = useForm(
+    initState,
+    submit,
+    validateUserForm);
+
+ 
+  return (
+    <Card
+    >
+      <form
+        onSubmit={handleSubmit}
+        autoComplete="off"
+        noValidate
+      >
+        <CardHeader
+          title="Nowy użytkownik"
+        />
+        <Divider />
+        <CardContent>
+          <Grid
+            container
+            spacing={3}
+          >
+            <Grid
+              item
+              md={12}
+              xs={12}
+            >
+            <Typography
+               gutterBottom
+               variant="h6">
+                 Podstawowe dane
+             </Typography>
+              </Grid>
+            <Grid
+              item
+              md={6}
+              xs={12}
+            >
+              <TextField
+                fullWidth
+                label="Imię"
+                margin="dense"
+                name="firstName"
+                onChange={handleChange}
+                required
+                value={values.firstName}
+                variant="outlined"
+              />
+              <p className="error">{errors.firstName}</p>
+            </Grid>
```

Comment:
I think we should stick to one style convention for  JSX tags, above we had e.g. 
`<CardHeader `
`title="Nowy użytkownik"  ` 
`/>`